packages/terra-date-input/src/DateInputField.jsx

```diff
@@ -136,6 +141,7 @@ const defaultProps = {
   showOptional: false,
   value: '',
   yearAttributes: {},
+  errorMessageId: null,
```

Comment:
Same here, where is `errorMessageId` being used

Reply:
This is removed as per latest changes.